application/tests/codeception/config/functional.php

```diff
@@ -1,18 +1,24 @@
 <?php
 /**
- * Application configuration shared by all applications functional tests
+ * Application configuration shared by all applications unit tests
```

Comment:
Looks like this may have been an accidental change, since this is the `functional.php` config file.